### PR TITLE
Software in nav reimagine2

### PR DESCRIPTION
**Pull request by niuage**

### CODE REVIEW

**Comment by niuage** [comment]
[(1) Move Projects link to top nav](https://trello.com/c/Wp5M6Ntg/4908-1-move-projects-link-to-top-nav)

File: app/assets/javascripts/reimagine2/mobile_global_nav.coffee [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@Reimagine2.mobileGlobalNav = ->
+  # $(".toggle-topbar").on()
```
Comment (devpost-mzheng) [inline app/assets/javascripts/reimagine2/mobile_global_nav.coffee]:
```suggestion
```

File: app/views/reimagine2/devpost/_global_nav_mobile.html.erb [hunk 2]
```diff
@@ -1,19 +1,29 @@
 <div class="contain-to-grid hide-for-large-up">
   <nav class="top-bar" data-topbar>
     <ul class="title-area top-bar-section">
+      <li>
+        <%= link_to "#",
+                    class: "toggle-topbar has-dropdown text-center",
+                    data: { target_menu: "world" },
+                    id: "hamburger" do %>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/_global_nav_mobile.html.erb]: hamburgers don't have ham in them

File: app/assets/javascripts/reimagine2/mobile_global_nav.coffee [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  $(document).on "click.fndtn.topbar", '.top-bar .toggle-topbar, [data-topbar] .toggle-topbar', (e) ->
```
Comment (devpost-mzheng) [inline app/assets/javascripts/reimagine2/mobile_global_nav.coffee]: any reason for the multiple selectors? they get the same element dont they?

Comment (devpost-mzheng) [reply app/assets/javascripts/reimagine2/mobile_global_nav.coffee]: i see this is just the selector they use

File: app/views/reimagine2/devpost/_global_nav_mobile.html.erb (hunk 2 above)
Comment (niuage) [reply app/views/reimagine2/devpost/_global_nav_mobile.html.erb]: https://www.google.com/url?sa=i&source=images&cd=&cad=rja&uact=8&ved=2ahUKEwjDlLDmzf_hAhVDneAKHQMFCQUQjRx6BAgBEAU&url=https%3A%2F%2Fknue.com%2Ftoday-is-fun-facts-about-your-name-day%2F&psig=AOvVaw1sceZHOvKGkLq6XcCVL-20&ust=1556981380805927

**Pull request merged**